### PR TITLE
add collectd::plugin::ceph

### PR DESCRIPTION
**Pull request by TheMeier**
Class to manage ceph plugin for collectd

### CODE REVIEW

**Comment by blkperl** [comment]
Please update the README with docs and add a quick spec test.

**Comment by TheMeier** [comment]
added basic test and docs

File: spec/classes/collectd_plugin_ceph_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+</Plugin>
+EOS
+    it 'Will create /etc/collectd.d/10-ceph.conf' do
+      should contain_collectd__plugin('ceph').with_content(content)
+      #should contain_file('ceph.load').with({
+      #  :ensure  => 'present',
+      #  :path    => '/etc/collectd.d/10-ceph.conf',
+      #  :content => content,
+      #})
```
Comment (blkperl) [inline spec/classes/collectd_plugin_ceph_spec.rb]: Did you mean to leave commented code here?

Comment (TheMeier) [inline spec/classes/collectd_plugin_ceph_spec.rb]: nope, fixed...

**Comment by blkperl** [comment]
Great, thanks for all the work!

**Pull request merged**